packages/block-library/src/post-title/edit.js

```diff
@@ -30,7 +30,19 @@ export default function PostTitleEdit( {
 } ) {
 	const TagName = 0 === level ? 'p' : 'h' + level;
 	const isDescendentOfQueryLoop = Number.isFinite( queryId );
-	const userCanEdit = useCanEditEntity( 'postType', postType, postId );
+	/**
+	 * Hack: useCanEditEntity may trigger an OPTIONS request. When the post title
+	 * is a descendant of a query loop, we know for sure that user cannot edit
+	 * the title. To avoid unnecessary OPTIONS requests we should avoid using
+	 * useCanEditEntity. However, it's a hook and cannot be fired conditionally,
+	 * hence instead we call it without proper data. In this case hook automatically
+	 * returns false without calling a backend.
+	 */
+	const userCanEdit = useCanEditEntity(
+		'postType',
+		isDescendentOfQueryLoop ? '' : postType,
```

Comment:
I think just a `! isDescendentOfQueryLoop && postType` would be enough and we could always pass the `postId` for minimum changes.

Reply:
Makes sense! Change applied in [commit](https://github.com/WordPress/gutenberg/pull/49839/commits/76bb969e55963c74976394629079440da66a9308)